libs/@hashintel/design-system/src/icon-download.tsx

```diff
@@ -0,0 +1,20 @@
+import { SvgIcon, SvgIconProps } from "@mui/material";
+import { FunctionComponent } from "react";
+
+export const DownloadIcon: FunctionComponent<SvgIconProps> = ({
```

Comment:
Can we include the variant of this FA icon in the component name? (e.g. `DownloadRegularIcon` if it is `regular`). In the designs we are using both `light` and `regular` icons in various places, so this prevents potential clashes between icon names down the line.

I think this also applies to the other icons introduced in this PR.

Reply:
Done in https://github.com/hashintel/hash/pull/3216/commits/ca72562a79998eb2dce24f7f67b9908d09b55940